tsconfig.json

```diff
@@ -0,0 +1,40 @@
+{
+  "compilerOptions": {
+    "module": "commonjs",
+    "preserveConstEnums": true,
+    "sourceMap": true,
+    "newLine": "LF",
+    "target": "es6",
```

Comment:
if you want node 8 to be your min version, make this "es2017". you'll get native async/await and some other goodies to improve productivity/perf